packages/patrol_cli/lib/src/test_bundler.dart

```diff
@@ -30,10 +30,16 @@ ${generateImports(testFilePaths)}
 // END: GENERATED CODE
 
 Future<void> main() async {
-  // TODO: Create and use PatrolNativeTestService instead of NativeAutomator
   final nativeAutomator = NativeAutomator(config: NativeAutomatorConfig());
   final binding = PatrolBinding.ensureInitialized();
-  // Create a PatrolAppService.
+  
+  // This is the entrypoint of the bundled Dart test.
```

Comment:
@shilangyu I'd be happy if you reviewed this comment.

Reply:
Is this generated `test_bundle` expected to be committed by users? 

These docs are nice, but I wonder if its a good place for them. Will anyone care what this file does exactly? Seems like an implementation detail.

One thing I dont exactly understand are the `// START: GENERATED CODE` and `// END: ...`. The whole file is generated. Maybe add a standard header of `// GENERATED CODE - DO NOT MODIFY BY HAND`? 

Also, instead of disabling specific lint rules (which assumes this code wont generate warnings for all linter configurations) you can disable them all with `// ignore_for_file: type=lint`.

Reply:
> Is this generated `test_bundle` expected to be committed by users?

No, it's not. I'll have to make it more clear by putting something like `// GENERATED CODE - DO NOT COMMIT INTO VCS`.

Actually, at first, I tried generating it in `.dart_tool/patrol_build`, but _very strange_ things happened (tests that were not referenced by `test_bundle` were being executed). I should probably try doing that again. 

> I wonder if it's a good place for them. Will anyone care what this file does exactly? Seems like an implementation detail.

I share these doubts, but I think that if these docs were in a different place than the code, it'd be easier for them to drift away over time.

Also, I expect almost nobody to bother looking at that file. And if somebody does, in the worst case they won't understand anything and close it, and in the best, they'll learn about how Patrol works :)

> One thing I dont exactly understand are the `// START: GENERATED CODE` and `// END: ...`. The whole file is generated. Maybe add a standard header of `// GENERATED CODE - DO NOT MODIFY BY HAND`?

The whole file is generated, but it's mostly static. The 2 parts surrounded with `// START: GENERATED CODE` are actually generated by walking the hierarchy of the `integration_test` directory.

> you can disable them all with `// ignore_for_file: type=lint`

Thanks - I unknowingly assumed the only ruleset in the world is 🖤leancode_lint💛.


I created PR #1283 to apply these suggestions.